ci/copy-windows-release-artifacts.sh

```diff
@@ -11,7 +11,7 @@ mkdir -p $OUTPUT_DIR/github
 mkdir -p $OUTPUT_DIR/artifactory
 mkdir -p $OUTPUT_DIR/split-release
 
-if [ "$SPLIT_RELEASE" = "false" ]; then
+if ! [ "$SPLIT_RELEASE" = "true" ]; then
```

Comment:
If we were dealing with booleans this would be a meaningless (or meaning-preserving, I guess) change, but I suspect (yes, without proof and with low confidence) our current failure is specifically because we do not end up with a boolean value in `$3`.